content/en/blog/2020-11-02-charts-repo-deprecation.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Helm Chart Repository Deprecation Update"
+slug: "charts-repo-deprecation"
+authorname: "Vic Iglesias"
+author: "@vicnastea"
+authorlink: "https://twitter.com/vicnastea"
+date: "2020-11-02"
```

Comment:
Can you update this for Friday's date?

Reply:
Apologies had forgotten to push the change